docs/mkdocs/docs/api/basic_json/meta.md

```diff
@@ -11,14 +11,16 @@ the platform and compiler.
 
 JSON object holding version information
 
-| key         | description                                                                                                                                                                                                                                                                    |
-|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `compiler`  | Information on the used compiler. It is an object with the following keys: `c++` (the used C++ standard), `family` (the compiler family; possible values are `clang`, `icc`, `gcc`, `ilecpp`, `msvc`, `pgcpp`, `sunpro`, and `unknown`), and `version` (the compiler version). |
-| `copyright` | The copyright line for the library as string.                                                                                                                                                                                                                                  |
-| `name`      | The name of the library as string.                                                                                                                                                                                                                                             |
-| `platform`  | The used platform as string. Possible values are `win32`, `linux`, `apple`, `unix`, and `unknown`.                                                                                                                                                                             |
-| `url`       | The URL of the project as string.                                                                                                                                                                                                                                              |
-| `version`   | The version of the library. It is an object with the following keys: `major`, `minor`, and `patch` as defined by [Semantic Versioning](http://semver.org), and `string` (the version string).                                                                                  |
+| key         | description                                                                                                                              |
+|-------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| `compiler`  | Information on the used compiler. It is an object with the following keys: `c++` (the used C++ standard), `family` (the compiler family; |
+|             | possible values are `clang`, `icc`, `gcc`, `ilecpp`, `msvc`, `pgcpp`, `sunpro`, and `unknown`), and `version` (the compiler version).    |
```

Comment:
Putting this in a new row looks odd.

Reply:
Odd how?
A markdown table with line wrapping is borderline unreadable. Shorter lines => less chance of line wrapping => more readable table.

Reply:
In GitHub's Markdown rendering, it looks odd to have empty cells:

<img width="1031" alt="image" src="https://user-images.githubusercontent.com/159488/179395939-6745c4a9-f91e-4d36-b7d5-f51d7b41652c.png">

Is it the same in mkdocs?

Reply:
Oh, I did not know that. Let me check.

Reply:
There is an extension to markdown known as grid tables but it's not supported by python-markdown. ;-(

I'll revert the table changes.